config.yaml

```diff
@@ -59,10 +65,8 @@ submodels:
       input:
         - /g/data/vk83/experiments/inputs/access-om2/ice/grids/global.01deg/2020.05.30/grid.nc
         - /g/data/vk83/experiments/inputs/access-om2/ice/grids/global.01deg/2020.05.30/kmt.nc
-        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions_biogeochemistry/global.01deg/2022.02.24/o2i.nc
-        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions_biogeochemistry/global.01deg/2022.02.24/i2o.nc
-        - /g/data/vk83/experiments/inputs/access-om2/ice/initial_conditions/global.01deg/2020.05.30/monthly_sstsss.nc
```

Comment:
Sorry to be a pain, but these should remain. They are in the 1 degree config

https://github.com/ACCESS-NRI/access-om2-configs/blob/release-1deg_jra55_ryf_bgc/config.yaml#L69-L70

When I asked Dougie to delete the `o2i.nc` and `i2o.nc` files from the 1 degree it was because they were duplicates from the non-bgc config https://github.com/ACCESS-NRI/access-om2-configs/pull/60#discussion_r1555202578

Reply:
Done here. [9360475](https://github.com/ACCESS-NRI/access-om2-configs/pull/63/commits/93604751aeebeeee2ff63b15af8bdae32db9671d)